highway_env/vehicle/behavior.py

```diff
@@ -98,6 +99,10 @@ def act(self, action: Union[dict, str] = None):
 
         # Longitudinal: IDM
         front_vehicle, rear_vehicle = self.road.neighbour_vehicles(self, self.lane_index)
+        # Parked vehicles are ignored for regulation rules (not collision avoidance).
```

Comment:
I am not sure I understand the use case: you can collide with a parked vehicle, but you don't slow down and stop if there is a parked vehicle on your lane?

Reply:
Sorry for the late reply. 
We had the case that vehicles react to parked (off-road) cars since they are matched onto the lane. This quick-fix enables to tag vehicles as parked and avoid reaction of others to those cars. In our case, collision avoidance is not relevant since parked cars are off-road. A more sophisticated fix might be to encounter lateral deviation in the behavior action directly.
Nevertheless, the comment needs to be changed as it affects both collision avoidance and regulatory rules.